components/configuring_network_bridge.mdx

```diff
@@ -0,0 +1,46 @@
+Your is configured to bridge its physical ethernet ports together. This allows any ethernet port to be used
```

Comment:
- Change to 'Your computer is configured...'
- Change to 'Ethernet' or 'network'   (2 instances)

Reply:
Done